test/fabric/src/main/resources/mods.groovy

```diff
@@ -54,8 +54,6 @@ final mdg = FabricModsDotGroovy.make {
         java = ">=17"
         it.'fabric-api' = "*"
 
-        mod "another-mod", ">=1.5.0"
-        mod("something-else", v(">0.5") & v('<1.0'))
```

Comment:
I would like a version range combination test like this to remain in the file still